Microsoft.Toolkit.Uwp.UI.Controls.Markdown/Render/MarkdownRenderer.Inlines.cs

```diff
@@ -257,7 +257,7 @@ protected override async void RenderImage(ImageInline element, IRenderContext co
             var viewbox = new Viewbox
             {
                 Child = hyperlinkButton,
-                StretchDirection = StretchDirection.DownOnly
```

Comment:
Did we have our own `StretchDirection` property instead of using the system one?

Reply:
Yeah, the `.Uwp.UI` package has another `StretchDirection` which was causing a conflict here since it was already in scope, so I had to use the fully qualified name. It's this one here: https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/StretchDirection.cs.

Reply:
Since the platform exposes a `StretchDirection` property, should we change our toolkit's enum for ListViewBase? Like `StretchItemDirection` or `StretchItemContainerDirection` to match the property name?

Reply:
Discussed with Sergio, let's rename both property and enum to `ItemContainerStretchDirection` to align with the base `ItemContainerStyle` which is the alternate approach as well as the `ItemContainerContentChanging` event.

Reply:
Fixed in 435c96debde1146318bf1df88b2dba37f79ccddb.